src/Writer/XmlWriter.php

```diff
@@ -80,6 +84,9 @@ public function close(): void
         fclose($this->file);
     }
 
+    /**
+     * @throws SonataExporterException
```

Comment:
You don't need this, when you add this to the interface

Reply:
You should also wrap the existing `RuntimeException` inside the `SonataExporterException`

Reply:
What do you mean by the second message ?

Reply:
Ok, I create a generic `class RuntimeException extends \RuntimeException implements SonataExporterException`.

This way when creating a custom writer it's easier to do
```
    public function open()
    {
        try {
            $this->myWriterFromExtLibrary->openButCanThrowException();
        } catch (\Exception $e) {
            throw new Sonata\Exporter\Exception\RuntimeException()
        }
    }
```
without having to create a custom SonataExporterException too